src/sentry/services/hybrid_cloud/organization_actions/impl.py

```diff
@@ -33,7 +34,7 @@ def create_organization_and_member_for_monolith(
     organization_name,
     user_id,
     slug: str,
-) -> Tuple[Organization, OrganizationMember]:
+) -> Tuple[Organization, OrganizationMember, Team]:
```

Comment:
Once a tuple reaches 3, I ponder if a new RpcModel makes more sense

Reply:
err, dataclass

Reply:
Fair! I'll add one for this